scripts/mattermost-postgresql-5.0.sql

```diff
@@ -398,7 +398,7 @@ CREATE TABLE public.posts (
     props character varying(8000),
     hashtags character varying(1000),
     filenames character varying(4000),
-    fileids character varying(150),
+    fileids character varying(300),
```

Comment:
It's not strictly needed but since we are here we could also update `scripts/mattermost-mysql-5.0.sql`.